stdlib/LibGit2/src/config.jl

```diff
@@ -89,23 +89,26 @@ function addfile(cfg::GitConfig, path::AbstractString,
                  repo::Union{GitRepo, Nothing} = nothing,
                  force::Bool=false)
     ensure_initialized()
-    @static if LibGit2.VERSION >= v"0.27.0"
-        @check ccall((:git_config_add_file_ondisk, libgit2), Cint,
-                     (Ptr{Ptr{Cvoid}}, Cstring, Cint, Ptr{Cvoid}, Cint),
-                     cfg.ptr, path, Cint(level), isa(repo, GitRepo) ? repo.ptr : C_NULL, Cint(force))
-    else
-        repo === nothing || error("repo argument is not supported in this version of LibGit2")
-        @check ccall((:git_config_add_file_ondisk, libgit2), Cint,
-                     (Ptr{Ptr{Cvoid}}, Cstring, Cint, Cint),
-                     cfg.ptr, path, Cint(level), Cint(force))
+    GC.@preserve cfg repo begin
+        # FIXME Check first argument, cfg.ptr, in the ccall's below.
```

Comment:
Probably better to define the unsafe_convert?

Reply:
ok but I was not sure whether `Ptr{Ptr{Cvoid}}` wass correct here, this was the meaning of the `FIXME` comment. Note that there is a similar issue with `GitStatus` object in `status.jl`, line 24.

Reply:
Looks like it should be Ptr{Cvoid}

Reply:
You are right. I'll fix that.